templates/docs/patterns/icons/index.md

```diff
@@ -131,7 +131,14 @@ If you use a limited set of icons you may want to include them individually to r
 @include vf-p-icons-common;
 
 // include only the icons that you use in your project
-{% for icon in standard_icons + status_icons + social_icons %}@include vf-p-icon-{{ icon }};
+{% for icon in standard_icons + status_icons + social_icons if icon != 'chevron-down' -%}
```

Comment:
As discussed in review before, would it be possible to move the logic from here somewhere to where lists of icons are created, and maybe create a separate list of `icon_mixins` to iterate through here?

Reply:
@bartaz 

Revised. Now I create a list of `base_icon_mixin_names` (contains the standard, status, & social icons). I didn't combine them all into a single `icon_mixins` list as the "additional icons" are visually separated from the rest in the import documentation and have a comment separately inserted, so I wanted to keep that in place.